src/app/f2f/steps.js

```diff
@@ -73,7 +73,7 @@ module.exports = {
       },
     ],
   },
-  "/ukPassportDetails": {
+  "/uk-passport-expire": {
```

Comment:
Is it a requirement for the path to be this? Was thinking that `uk-passport-expiry` is probably better grammar and more consistent

Reply:
This was in line with the F2F build review report from design